src/routes/api/tripRoute.js

```diff
@@ -480,7 +486,40 @@ tripRouter
   .get(
     '/stats/:tripTypeId',
     verifyToken, Validate.tripStatsRules(),
-    checkInputDataError, ValidateTrip.dateValidation, CustomValidator.isUserOrManager, getTripStatsController
+    checkInputDataError,
+    ValidateTrip.dateValidation,
+    CustomValidator.isUserOrManager,
+    getTripStatsController
+  );
+/**
+ * @swagger
+ *
+ * /trips/most-traveled:
+ *    get:
+ *      summary: get info about most traveled destination
+ *      tags: [Trip]
+ *      parameters:
+ *        - name: token
+ *          in: header
+ *          description: enter token
+ *          required: true
+ *          schema:
+ *            type: string
+ *      responses:
+ *        "200":
+ *          description:  Most traveled destination info retrieved successfully
+ *        "401":
+ *          description: No token provided
+ *        "404":
+ *          description: No Trip found
```

Comment:
Do we really need this?
I mean this 404 error?

Reply:
I removed it @victorkarangwa4